src/routes/ExercisesScreens.tsx

```diff
@@ -94,10 +94,14 @@ const ExercisesScreen = ({ route, navigation }: ExercisesScreenPropsType): JSX.E
   }
 
   const handleNavigation = (item: ExerciseType): void => {
-    setSelectedKey(item.key.toString())
-    navigation.navigate(EXERCISES[item.key].nextScreen, {
-      discipline: discipline
-    })
+    if (item.title === 'Level 1' && numberOfChildren < 4) {
+      Alert.alert('Not enough words')
```

Comment:
Please also don't use this here as a string directly, but add a entry to the labels file, with a speaking name. German translation for the alert message should be: 'Die Übung ist für dieses Modul nicht möglich (zu wenig Wörter)'